source/system-administrators/activities/reindexing-es/reindex-3-1-10-to-3-1-17.rst

```diff
@@ -0,0 +1,125 @@
+:is-up-to-date: True
+
+:orphan:
+
+.. document does not appear in any toctree, this file is referenced
+   use :orphan: File-wide metadata option to get rid of WARNING: document isn't included in any toctree for now
+
+.. _reindex-content-for-3-1-10-to-3-1-17:
+
+--------------------------------------------------------------------------------
+Reindexing Content Using Elasticsearch for Crafter CMS versions 3.1.10 to 3.1.17
+--------------------------------------------------------------------------------
+
+The steps listed below for reindexing content without disrupting service in production using Elasticsearch applies only to Crafter CMS versions 3.1.10 to 3.1.17
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 1: Create a new index
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to create a new index on Elasticsearch where you can index the content:
+
+#. Check the next version for the index name, you can find the current name using the following command:
+
+   .. code-block:: bash
+      :linenos:
+
+          curl -s http://ES_HOST:ES_PORT/_cat/aliases | grep SITE_NAME
+
+      # Example result
+
+      SITE_NAME   SITE_NAME_v1   - - -
```

Comment:
I think the whole doc need to be updated to include the environment too, both the index and alias include it:
```
SITE_NAME-ENV    SITE_NAME-ENV_vX
```